build/config/SignToolData.json

```diff
@@ -111,6 +111,7 @@
         "Microsoft.CodeAnalysis.Elfie.dll",
         "Microsoft.DiaSymReader.dll",
         "Microsoft.DiaSymReader.Native.amd64.dll",
+        "Microsoft.DiaSymReader.Native.arm.dll",
```

Comment:
>"Microsoft.DiaSymReader.Native.arm.dll", [](start = 8, length = 40)

Why was this entry needed?  Generally it means that we are putting this DLL into the VSIX where we didn't previously.